HelloMixpanel/HelloMixpanelTests/MixpanelOptOutTests.m

```diff
@@ -46,7 +51,7 @@ - (void)testOptOutTrackingWillNotGenerateEventQueue
 {
     stubTrack();
     [self.mixpanel optOutTracking];
-    
+    [self waitForMixpanelQueues];
```

Comment:
i think it's actually better to not wait for the queues in this and the following tests that are for events/people stuff. so that it'll actually also test that it works just by calling opt out and then tracking after the way users would do it